<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>0</y>
        <width>445</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>알람을 끄고 싶다면 달걀을 깨세요 ^-^</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>550</y>
        <width>241</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>0  / 10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="egg_lbl">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>35</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../UI_img/달걀.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>450</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>550</y>
        <width>70</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>G마켓 산스 TTF Bold</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
      </property>
      <property name="text">
       <string>닫기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../UI_img/지각.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>pushButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
